### PR TITLE
Fix downloading

### PR DESCRIPTION
**Pull request by liyuqian**
This would fix our device lab failure.

I didn't catch this mitake locally because I already downloaded the
resources...

### CODE REVIEW

File: packages/gauge/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## 0.1.1
+
+* Improve tests and remove unnecessary resources.
```
Comment (goderbauer) [inline packages/gauge/CHANGELOG.md]: Add "rename package to gauge" if it was already released under its old name?

File: packages/gauge/test/gauge_general_test.dart [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  final String gaugeRootPath = Directory.current.absolute.path;
+
+  test('help works', () {
+    final ProcessResult result = Process.runSync(
+      'dart',
+      <String>['$gaugeRootPath/bin/gauge.dart', 'help'],
+    );
+
+    expect(
+        result.stdout.toString(),
```
Comment (goderbauer) [inline packages/gauge/test/gauge_general_test.dart]: nit: add a trailing comma after container(...) and format it like the other expects below.

File: packages/gauge/test/gauge_ios_test.dart [hunk 3]
```diff
@@ -8,30 +8,18 @@ import 'dart:io';
...
-import 'package:measure/commands/base.dart';
+import 'package:gauge/commands/base.dart';
 
 void main() {
-  const String measureRootPath = '.';
+  final String gaugeRootPath = Directory.current.absolute.path;
   final String resourcesRootPath = BaseCommand.defaultResourcesRoot;
-  BaseCommand.doEnsureResources(resourcesRootPath, isVerbose: true);
-
-  test('help works', () {
```
Comment (goderbauer) [inline packages/gauge/test/gauge_ios_test.dart]: Why remove this test?

**Comment by goderbauer** [comment]
Can you add a test that would have failed without the await? To make sure we never regress this?

File: packages/gauge/test/gauge_ios_test.dart (hunk 3 above)
Comment (liyuqian) [reply packages/gauge/test/gauge_ios_test.dart]: Sorry, the initial patch was full of irrelevant changes due to local branching error... I've now cleaned up the patch, and added a unit test.

**Pull request merged**